bin/spiced/Cargo.toml

```diff
@@ -21,7 +21,7 @@ tracing-subscriber.workspace = true
 metrics-exporter-prometheus = "0.13.0"
 
 [features]
-default = ["duckdb", "postgres", "sqlite", "mysql", "flightsql", "databricks", "dremio", "clickhouse"]
+default = ["duckdb", "postgres", "sqlite", "mysql", "flightsql", "databricks", "dremio", "clickhouse", "odbc"]
```

Comment:
Do we need to install odbc on our runners to support with as default feature or there is other way?